commit 3f472ab8a09d2dfc724fa667646ce47159d8c166
Author: wolfgar <stephan.rafin@laposte.net>
Date:   Thu Sep 25 02:50:27 2014 +0200

    Undef EGL_KHR_reusable_sync when imx support is enabled as
    egl*SyncKHR declarations will conflict

diff --git a/xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp b/xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp
index 1300a7f..c747118 100644
--- a/xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp
+++ b/xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp
@@ -79,23 +79,25 @@ static PFNEGLDESTROYIMAGEKHRPROC eglDestroyImageKHR;
 static PFNGLEGLIMAGETARGETTEXTURE2DOESPROC glEGLImageTargetTexture2DOES;
 #endif
 
-#if defined(EGL_KHR_reusable_sync)
-static PFNEGLCREATESYNCKHRPROC eglCreateSyncKHR;
-static PFNEGLDESTROYSYNCKHRPROC eglDestroySyncKHR;
-static PFNEGLCLIENTWAITSYNCKHRPROC eglClientWaitSyncKHR;
-#endif
-
 #ifdef HAS_IMXVPU
 #include "windowing/egl/EGLWrapper.h"
 #include "DVDCodecs/Video/DVDVideoCodecIMX.h"
 
 #define GL_VIV_NV12 0x8FC1
+#undef EGL_KHR_reusable_sync
 typedef void (GL_APIENTRYP PFNGLTEXDIRECTVIVMAPPROC) (GLenum Target, GLsizei Width, GLsizei Height, GLenum Format, GLvoid ** Logical, const GLuint * Physical);
 typedef void (GL_APIENTRYP PFNGLTEXDIRECTINVALIDATEVIVPROC) (GLenum Target);
 static PFNGLTEXDIRECTVIVMAPPROC glTexDirectVIVMap;
 static PFNGLTEXDIRECTINVALIDATEVIVPROC glTexDirectInvalidateVIV;
 #endif
 
+#if defined(EGL_KHR_reusable_sync)
+static PFNEGLCREATESYNCKHRPROC eglCreateSyncKHR;
+static PFNEGLDESTROYSYNCKHRPROC eglDestroySyncKHR;
+static PFNEGLCLIENTWAITSYNCKHRPROC eglClientWaitSyncKHR;
+#endif
+
+
 #if defined(TARGET_ANDROID)
 #include "DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.h"
 #endif
